Homework/src/by/tms/lesson12/additionalTasks/task_4/Runner.java

```diff
@@ -0,0 +1,33 @@
+package by.tms.lesson12.additionalTasks.task_4;
+
+import java.io.*;
+
+public class Runner {
+    /*
+    4) Сеарилизовать объект Автомобиль(Марка, скорость, цена). После
+    сериализации произвести обратный процесс
+     */
+    public static void main(String[] args) {
+        Car car = new Car("Porshe", 200, 100000);
+        ObjectOutputStream out = null;
+        try {
+            out = new ObjectOutputStream(new FileOutputStream("src/by/tms/lesson12/additionalTasks/task_4/output.dat"));
+            out.writeObject(car);
+            out.close();
```

Comment:
should be closed in finally block or instantiated in try-with-resources statement